docs/DeveloperGuide.md

```diff
@@ -268,60 +268,87 @@ _{Explain here how the data archiving feature will be implemented}_
+| `H`  | user                                       | add recipes                        | keep track of quantity of ingredients needed for a recipe                  |
+| `H`  | user                                       | delete an ingredient               | stop keeping track of the ingredient                                       | 
+| `H`  | user                                       | delete a recipe                    | stop keeping track of the recipe                                           |
+| `H`  | user                                       | find an ingredient by name         | locate details of ingredients without having to go through the entire list |
+| `H`  | user                                       | find a recipe by name              | locate details of recipes without having to go through the entire list     |
+| `H`  | user                                       | list all ingredients               | easily see all ingredients                                                 |
+| `H`  | user                                       | list all recipes                   | easily see all recipes                                                     |
+| `H`  | user                                       | sort ingredients by expiry date    | locate ingredients that are expired or expiring soon                       |
+| `H`  | user                                       | automatically update the inventory list after executing a recipe | update the ingredients I have used           |
+| `H`  | user                                       | view ingredient tags               | view details about the ingredient (ie. its expiry status)                  |
+| `H`  | user                                       | update ingredients                 | update the quantity of ingredients as needed                               |
+| `H`  | user                                       | update recipes                     | update the recipe as needed                                                |
+| `M`  | user                                       | import my saved database           | start using it again quickly                                               |
+
 
 *{More to be added}*
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is `Fridgy` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC01 - Delete ingredient**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  `User` requests to delete a specific ingredient by specifying its name, quantity and expiry date.
+2.  `Fridgy` deletes the ingredient.
```

Comment:
Should this be fully delete the item with matching name, quantity and expiry date only or can we adapt it to matching name and expiry date, then reduce the quantity. Delete item if quantity <= 0.

Reply:
I was thinking `update` does the same job as change or reduce quantity. Delete can actually simply just `delete index` to fully delete?

Reply:
In that case we can just delete by name and expiry date as each ingredient can be uniquely identified by those 2 properties. Or we can display the index for user to delete.

Reply:
Changes made